content/widgets.md

```diff
@@ -66,3 +66,9 @@
 	* Child view controller should be fine
 
 * If you have more than one widget, use the `UIApplicationShortcutWidget` Info-Plist key to specify the widget (by bundle ID) which should be shown upon 3D-touching the homescreen app icon.
+
+* Data Protection should not be enabled for widgets (meaning setting key ```com.apple.developer.default-data-protection``` to value a valid value in your widgets entitlements file). If you do, iOS will fail to create a snapshot of your widget after showing widgets in the lock screen. You can see a similar log entry in the console:
```

Comment:
(do not set `com.apple.developer.default-data-protection` to a valid value in your widget's entitlements file).
(use single ` (not three of them) for inline code format)

If you do, iOS will fail to create a snapshot of your widget on the lock screen. You will see a log entry in the console similar to this: